packages/react-native/ReactCommon/react/renderer/components/view/LayoutConformanceShadowNode.h

```diff
@@ -16,9 +16,13 @@ namespace facebook::react {
 constexpr const char LayoutConformanceShadowNodeComponentName[] =
     "LayoutConformance";
 
-using LayoutConformanceShadowNode = ConcreteShadowNode<
-    LayoutConformanceShadowNodeComponentName,
-    YogaLayoutableShadowNode,
-    LayoutConformanceProps>;
+class LayoutConformanceShadowNode final
+    : public ConcreteShadowNode<
+          LayoutConformanceShadowNodeComponentName,
+          YogaLayoutableShadowNode,
+          LayoutConformanceProps> {
+ public:
+  using ConcreteShadowNode::ConcreteShadowNode;
+};
```

Comment:
I don't think this part is required

Reply:
@javache Seems this check cannot work if we don't subclass ConcreteShadowNode:

https://github.com/facebook/react-native/blob/06751aa0d1750a66f9d6a0a13e9b61e30d9321bb/packages/react-native/ReactCommon/react/renderer/components/view/YogaLayoutableShadowNode.cpp#L505-L506

Reply:
Oh yeah, interesting. Maybe a different behaviour in compilers.